frontend/javascripts/oxalis/model/accessors/dataset_accessor.js

```diff
@@ -155,6 +161,19 @@ export class ResolutionInfo {
     return bestIndexWithDistance[0];
   }
 
+  hasSmallerAndOrHigherIndex(index: number): SmallerOrHigherInfo {
+    const indices = this.getAllIndices();
+    const hasSmallOrHigher = { smaller: false, higher: false };
+    indices.forEach(currentIndex => {
```

Comment:
I think, we usually use `for (const currentIndex of indices) { ... }` instead of the `forEach` method.